test/extended/core.sh

```diff
@@ -51,7 +51,6 @@ SKIP_TESTS=(
   DNS                     # Can't depend on kube-dns
   kube-ui                 # Not installed by default
   DaemonRestart           # Experimental mode not enabled yet
```

Comment:
If we don't mind https://github.com/kubernetes/kubernetes/issues/19023 and https://github.com/kubernetes/kubernetes/issues/17829 :)